merken/heartbeat.py

```
@@ -0,0 +1,352 @@
+        )
+        title = f"audit:should_heartbeat:tick{self._tick_count}:{ts}"
+        try:
+            self._mem._vstash.remember(  # noqa: SLF001
+                body,
+                title=title,
+                collection=AUDIT_COLLECTION,
+                layer=AUDIT_LAYER,
+            )
+        except Exception:
+            # Audit failures must never break the loop.
+            pass
```

Ruff (0.15.14)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)